runtime/vm/jvminit.c

```diff
@@ -8097,6 +8101,27 @@ setSignalOptions(J9JavaVM *vm, J9PortLibrary *portLibrary)
 		}
 	}
 
+#if defined(WIN32) && defined(_WIN32_WINNT_WINBLUE) && (_WIN32_WINNT_MAXVER >= _WIN32_WINNT_WINBLUE)
+	HMODULE h_kernel32 = GetModuleHandle(TEXT("kernel32.dll"));
+
+	if ((NULL != h_kernel32) && IsWindows8OrGreater()) {
```

Comment:
I don't think `HMODULE` is a pointer type which makes `NULL` inappropriate; use `(HMODULE)0` instead.

Reply:
Based on the information from https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types, I think `HMODULE` is a pointer.  

>  `HMODULE` A handle to a module. This is the base address of the module in memory. `HMODULE` and `HINSTANCE` are the same in current versions of Windows, but represented different things in 16-bit Windows. This type is declared in WinDef.h as follows: `typedef HINSTANCE HMODULE;`

```
typedef PVOID HANDLE;
typedef HANDLE HINSTANCE;
typedef HINSTANCE HMODULE;
```

Reply:
I stand corrected - thanks.